guide/additional-info/rest-api.md

```diff
@@ -13,21 +13,25 @@ To install node-fetch, run the following command:
 :::: code-group
 ::: code-group-item npm
 ```sh:no-line-numbers
-npm install node-fetch
+npm install node-fetch@2.6.5
 ```
 :::
 ::: code-group-item yarn
 ```sh:no-line-numbers
-yarn add node-fetch
+yarn add node-fetch@2.6.5
 ```
 :::
 ::: code-group-item pnpm
 ```sh:no-line-numbers
-pnpm add node-fetch
+pnpm add node-fetch@2.6.5
 ```
 :::
 ::::
 
+::: tip
+We're installing version 2.6.5 of `node-fetch` because versions 3+ do not support `require()` syntax.
```

Comment:
Wrong "we" usage. You're not referring to the guide writers specifically here (we aren't installing anything on the reader's system). I'd propose

```suggestion
Newer versions of `node-fetch` do not support the [CommonJS](https://nodejs.org/api/modules.html#modules_modules_commonjs_modules) `require()` syntax.
```

...or similar.